SwaprHQ/stackly-ui
# [STK-31][FEAT] - TokenPicker kick-off work

## Fixes: [STK-31](https://linear.app/swaprdev/issue/STK-31/create-tokenpicker)

# Description

* Remove unneeded elements
* Refactor and abstract components into smaller pieces, applied re-styling to many components
* Create a few small utility components, they're dummies for now
* Add a few safety checks and default image fallbacks
* Connect the current `tokenList` for the `TokenPicker` component to consume and display

![image](https://github.com/SwaprHQ/stackly-ui/assets/21271189/dbc0d73b-64fe-44d8-8a7a-80c6aea5e4cd)


# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Stackly landing page, click any "select token" button and wait for the modal to pop-up
4) Modal should look similar to [these designs](https://www.figma.com/file/XsJ1Cwv46rhwVOWzlAdhhH/Stackly-V1.0?type=design&node-id=602%3A22611&mode=dev)
